app.py

```diff
@@ -1,55 +1,124 @@
-from flask import Flask, request
-from flask_restful import reqparse, abort, Api, Resource
-from flask_pymongo import PyMongo
-
+from flask import Flask
+from flask_restful import Resource, Api, reqparse, request, abort
+from model import Model
+from voluptuous.schema_builder import Schema, Required
+from voluptuous.validators import Length, All, MultipleInvalid
+import json
 app = Flask(__name__)
-mongo = PyMongo(app)
 api = Api(app)
 
-
-USERS = {
-    'user1': {'name': 'elena'},
-    'user2': {'name': 'bob'},
-}
-
-def abort_if_user_doesnt_exist(user_id):
-    if user_id not in USERS:
-        abort(404, message="User {} doesn't exist".format(user_id))
-
 parser = reqparse.RequestParser()
 parser.add_argument('name')
 
-class Users(Resource):
-    def get(self, user_id):
-        abort_if_user_doesnt_exist(user_id)
-        return USERS[user_id]
-
-    def delete(self, user_id):
-        abort_if_user_doesnt_exist(user_id)
-        del USERS[user_id]
-        return '', 204
+idSchema = Schema(All(str, Length(min=24, max=24)))
+jsSchema=Schema(str)
 
-    def put(self, user_id):
+class Student(Resource):
+    def get(self, student_id):
+        try:
+            if(idSchema(student_id)==student_id):
+                student=Model.get_by_id(student_id)
+            else:
+                raise AssertionError('MultipleInvalid not raised')
+        except MultipleInvalid as e:
+            exc = e
+            print (str(exc))
+            return '', 406
```

Comment:
`400` .. not 406 .. that has to deal with the Headers Accept-Language not matching what the server is responding with .. for example `application/json` is expected but you are return xml